data/db/mil-tamron.xml

```diff
@@ -1,6 +1,109 @@
 <!DOCTYPE lensdatabase SYSTEM "lensfun-database.dtd">
 <lensdatabase version="2">
 
+    <lens>
+        <maker>Tamron</maker>
+        <model>E 17-70mm F2.8 B070</model>
+        <mount>Sony E</mount>
+        <cropfactor>1.5</cropfactor>
```

Comment:
The new version of the database typically has more accurate crop factors where possible. Can I ask which camera you took the calibration photos with?

Reply:
I'll update the xml with the camera name too, it got lost when I updated the commit. 

I took the photos with a sony a6000.
Which crop factor should I use? 1.53?

Reply:
It looks like 1.534 is the value that others have used from that camera, so I'd go with that.

Reply:
I see that you already have. Thanks.